lib/Mojolicious/Plugin/DefaultHelpers.pm

```diff
@@ -189,7 +189,7 @@ sub _proxy_start_p {
       my $write = $source_content->is_chunked ? 'write_chunk' : 'write';
       $source_content->unsubscribe('read')->on(
         read => sub {
-          $content->$write(pop) and $tx->resume;
+          $content->$write(length $_[1] ? $_[1] : undef) and $tx->resume;
```

Comment:
We only use `$_[x]` directly for one-liners and performance optimisations. Is this a performance optimisation?

Reply:
No, I was attempting to imitate the "code-golfing" style. Would you prefer me to make it into `my $data = pop` above?

Reply:
In anticipation that you do in fact want that, and hopefully save you an iteration, I've adjusted it and force-pushed. Please advise if in fact you'd like something different.